includes/Api/Metrics.php

```
@@ -0,0 +1,248 @@
+    /**
+     * Get monthly usage for a specific month
+     *
+     * Uses DataLayer to count sent SMS for a specific month up to a given day.
+     *
+     * @param SmsStatStore $store Store instance
+     * @param string       $month Month in 'Y-m' format
+     * @param int          $day   Day of month for comparison
+     *
+     * @return int Total sent messages
+     */
+    private function get_monthly_usage( $store, $month, $day ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$store'. (undefined)

(UnusedFormalParameter)